lib/reporting/identity_verification_report.rb

```diff
@@ -218,11 +229,41 @@ def usps_enrollment_status_updated
       data[Events::USPS_ENROLLMENT_STATUS_UPDATED].count
     end
 
+    def passed_fraud_review_users
+      # Fraud review events may not be tagged with the issuer.
+      # When we are filtering by SP, we only count fraud review events where
+      # there is another event for the user in the data that _is_ tagged
+      # with the issuer.
+
+      result = data[Events::FRAUD_REVIEW_PASSED]
+
+      issuers&.each do |issuer|
+        users_with_events_for_issuer = data[sp_key(issuer)]
+        result &= users_with_events_for_issuer
```

Comment:
Confession: I have _never_ seen `&=` in the wild before this, and [had to look it up](https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Operators#Operators) to refresh my memory. If not for the comment I'm not sure I would understand what this was doing.

Reply:
I had to open irb to remember what `&` and `|` did with sets

I'm _pretty_ sure the linter would've complained at `result = result & user_with_events_for_issuer`

Reply:
I was just looking over this and I was wondering what happens if we have multiple issuers. Am I reading this correctly that it filters it down to users who have events for both issuers?

I'd expect we'd compute the set all relevant issuers and then do the intersection between fraud review passed users and that set instead of doing it by issuer.

Reply:
Yeah Hooper I think you're right

Reply:
Addressed in [fe02919](https://github.com/18F/identity-idp/pull/11311/commits/fe02919bf0316900e27fc065c1129f70af83f944), I just need to write a spec and then I think this can land

Reply:
Ok spec added in [ef6b228](https://github.com/18F/identity-idp/pull/11311/commits/ef6b228f7192a09b8e6ac9b66b8a377b2c2db87a), and I confirmed that it fails prior to the change in fe02919